### ISSUE TITLE
openvr_driver.h is outdated

### ISSUE THREAD

**Issue by sl1pkn07** [ChristophHaag/SteamVR-OpenHMD#81]
Hi

the header openvr_driver.h in the subproject folder is outdated

updated:
https://raw.githubusercontent.com/ValveSoftware/openvr/master/headers/openvr_driver.h

greetings

**Comment by thaytan** [comment]
True, but there's also no need to update except to use new features only present in the newer header. The driver pulls in version numbers from the header it's compiled against too - so if the header gets updated, the driver code needs to get updated to make sure it supports anything that changed

**Comment by sl1pkn07** [comment]
with "driver" you mean?

greetings

**Comment by thaytan** [comment]
I mean this SteamVR-OpenHMD driver

**Comment by sl1pkn07** [comment]
hi

yes, thats is the idea

greetings
